### ISSUE TITLE
Float project prices cause problems

### ISSUE THREAD

**Issue by martinlackner** [pbvoting/pabutools#5]
When project prices are floats, some functionality breaks. I have in particular encountered issues with gmpy2 (fractions can't be initialized with floats). 

Easy solution: require costs to be integers.

**Comment by Simon-Rey** [comment]
Not if you first pass them to frac(cost) right?

`from pabutools.fraction import frac

cost = frac(2.4864)
p.cost = cost`

Should work

**Comment by martinlackner** [comment]
Yes, that works. Would be nice if this is done automatically (if necessary).
